tldw_Server_API/tests/Sharing/conftest.py

```
@@ -0,0 +1,69 @@
+
+    async def execute(self, sql: str, params: tuple = ()) -> None:
+        self._conn.execute(sql, params)
+        self._conn.commit()
+
+    async def fetchone(self, sql: str, params: tuple = ()) -> dict | None:
+        cur = self._conn.execute(sql, params)
+        row = cur.fetchone()
+        if row is None:
+            return None
+        cols = [d[0] for d in cur.description]
+        return dict(zip(cols, row))
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)